examples/importCommitments.ts

```diff
@@ -0,0 +1,45 @@
+import * as dotenv from "dotenv";
+import path from "path";
+import {
+  UserFactory,
+  BLOCKCHAIN_WS_URL_DEFAULT,
+  CLIENT_API_URL_DEFAULT,
+} from "../libs/user";
+
+const _rootPath = path.resolve();
+dotenv.config({ path: path.join(_rootPath, ".env") });
+
+const ETHEREUM_PRIVATE_KEY =
+  "0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69e";
+const NIGHTFALL_MNEMONIC = process.env.SDK_NIGHTFALL_MNEMONIC;
+
+const options = {
+  blockchainWsUrl: BLOCKCHAIN_WS_URL_DEFAULT,
+  clientApiUrl: CLIENT_API_URL_DEFAULT,
+  ethereumPrivateKey: ETHEREUM_PRIVATE_KEY,
+  nightfallMnemonic: NIGHTFALL_MNEMONIC,
+};
+
+/**
+ * @function main a script to test the import commitments flow
+ * @author luizoamorim
+ */
+const main = async () => {
+  let user;
+  try {
+    user = await UserFactory.create(options);
+
+    await user.importAndSaveCommitments(
+      "./",
+      "commitmentsBackup.json",
+      user.zkpKeys.compressedZkpPublicKey,
+    );
+  } catch (error) {
+    console.log(error);
+    process.exit(1);
+  } finally {
+    user.close();
+  }
+};
+
+main();
```

Comment:
Please update as per the latest changes in the other example scripts! Note the env vars management, inline comments, etc.

Reply:
Can we just add some more love to docs and comments? To keep it consistent with the other examples, please:

1 Drop JSDocs comment above

2 On top of line 10 - create user:
```
// # 1 Create an instance of User
```

3 On top of L17 - await import (see my other comment in `user.ts` file about passing params via `options` object):
```
// # 2 Import commitments from `pathToExport` in JSON
```